common/containers/Tabs/Swap/components/CurrencySwap.tsx

```diff
@@ -77,31 +82,39 @@ export default class CurrencySwap extends Component<
       destinationAmount
     );
 
-    if (disabled && originAmount && !this.state.showedMinMaxError) {
+    if (disabled && originAmount) {
       const { bityRates } = this.props;
       const ETHMin = generateKindMin(bityRates.BTCETH, 'ETH');
       const ETHMax = generateKindMax(bityRates.BTCETH, 'ETH');
       const REPMin = generateKindMin(bityRates.BTCREP, 'REP');
 
-      const notificationMessage = `
-        Minimum amount ${bityConfig.BTCMin} BTC,
-        ${toFixedIfLarger(ETHMin, 3)} ETH.
-        Max amount ${bityConfig.BTCMax} BTC,
-        ${toFixedIfLarger(ETHMax, 3)} ETH, or
-        ${toFixedIfLarger(REPMin, 3)} REP
-      `;
-
-      this.setState(
-        {
-          disabled,
-          showedMinMaxError: true
-        },
-        () => {
-          this.props.showNotification('danger', notificationMessage, 10000);
-        }
-      );
+      let minAmount;
+      let maxAmount;
+      switch (originKind) {
+        case 'BTC':
+          minAmount = toFixedIfLarger(bityConfig.BTCMin, 3);
+          maxAmount = toFixedIfLarger(bityConfig.BTCMax, 3);
+          break;
+        case 'ETH':
+          minAmount = toFixedIfLarger(ETHMin, 3);
+          maxAmount = toFixedIfLarger(ETHMax, 3);
+          break;
+        case 'REP':
+          minAmount = toFixedIfLarger(REPMin, 3);
+          break;
+        default:
+          console.log('Something went terribly wrong!');
```

Comment:
Got a console log here

Reply:
I think that's intentional, but to make that clearer I would use `console.warn` and be a little more descriptive in the message.